src/main/java/wooteco/subway/line/service/LineService.java

```diff
@@ -0,0 +1,62 @@
+package wooteco.subway.line.service;
+
+import java.util.List;
+import java.util.Optional;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import wooteco.subway.exception.DuplicateLineNameException;
+import wooteco.subway.exception.NotExistLineException;
+import wooteco.subway.line.domain.Line;
+import wooteco.subway.line.repository.LineRepository;
+
+@Service
+public class LineService {
+
+    private final LineRepository lineRepository;
+
+    public LineService(LineRepository lineRepository) {
+        this.lineRepository = lineRepository;
+    }
+
+    @Transactional
+    public Line createLine(Line line) {
+        if (lineRepository.isExistName(line)) {
+            throw new DuplicateLineNameException();
+        }
+        return lineRepository.save(line);
+    }
+
+    @Transactional
+    public Line showLine(Long id) {
+        Optional<Line> line = lineRepository.findById(id);
+        if (line.isPresent()) {
+            return line.get();
+        }
+        throw new NotExistLineException();
```

Comment:
이렇게 해도 될 것 같아요 :) 
```suggestion
        return lineRepository.findById(id)
                .orElseThrow(NotExistLineException::new);
```

Optional 관련된 링크도 남길게요! 
https://www.latera.kr/blog/2019-07-02-effective-optional/

Reply:
가독성이 더 좋은 방법 알려주셔서 감사합니다 😃